packages/libs/wdk-client/src/Service/ServiceBase.ts

```diff
@@ -350,14 +350,19 @@ export const ServiceBase = (serviceUrl: string) => {
       .getItem<ServiceConfig>('/__config')
       .then((storeConfig) => {
         if (storeConfig == null) {
-          return _fetchJson<ServiceConfig>('GET', '/').then((serviceConfig) => {
-            return _store.setItem('/__config', serviceConfig);
-          });
+          return fetch(serviceUrl)
+            .then((response) => response.json())
```

Comment:
I think we should catch/throw a failed request

```suggestion
            .then((response) => {
              if (!response.ok) {
                console.error(`Fetching ${serviceUrl} failed for _initializeStore: ${response.statusText}`);
                throw new Error("Failed to initialize service");
              }
              return response.json()
            })
```

Reply:
Great idea!